src/plugins/data/common/field_formats/mocks.ts

```diff
@@ -35,6 +36,9 @@ export const fieldFormatsMock: IFieldFormatsRegistry = {
   init: jest.fn(),
   register: jest.fn(),
   parseDefaultTypeMap: jest.fn(),
-  deserialize: jest.fn(),
+  deserialize: jest.fn().mockImplementation(() => {
+    const DefaultFieldFormat = FieldFormat.from(identity);
+    return new DefaultFieldFormat();
+  }),
```

Comment:
Added a mock implementation to improve these somewhat. I just stole this from the `getFormat` function inside of `fieldFormats.deserialize`.

Important to note that since this is configured with `identity`, any calls against this mock are just going to return whatever they are given. So you'll get a `convert` function you can use, but of course it won't actually convert what you give to it.